.travis.yml

```diff
@@ -1,14 +1,15 @@
 ---
-# Verify this with: http://lint.travis-ci.org/
 language: ruby
-# Delete dependency locks for matrix builds.
 before_install: rm Gemfile.lock || true
+bundler_args: --without functional_tests
```

Comment:
Any reason the commit message doesn't mention this change?

Reply:
Adding this was a mistake. Nice catch.